apps/web/vibes/2px/components/dropdown/index.tsx

```diff
@@ -0,0 +1,94 @@
+'use client'
+
+import { useState } from 'react'
+
+import clsx from 'clsx'
+
+import { ChevronDownIcon } from '../icons'
+
+interface OptionProps {
+  label: string
+  value: string
+}
+
+interface DropdownProps {
+  placeholder: string
+  options: OptionProps[]
+  status?: 'normal' | 'error' | 'success'
+  disabled?: boolean
+  value?: string
+  setValue: (value: string) => void
+  className?: string
+}
+
+const Dropdown = ({
+  placeholder,
+  options,
+  status = 'normal',
+  disabled,
+  setValue,
+  value,
+  className,
+}: DropdownProps) => {
+  const [isOpen, setIsOpen] = useState(false)
+
+  return (
+    <div className={clsx('relative w-full font-body text-foreground @container', className)}>
+      <button
```

Comment:
I think it'd be good to fix the height as 3.75rem and 5rem for small and big screens, respectively.

Reply:
It's actually not @freadbaker , let's say someone uses this with another vibe where that line-height is bigger. having a fixed height is bad in that case.
As a rule of thumb if you can keep the height or width responsive do it.